### DISCUSSION TITLE
Deprecation/removal of VMDIRAC

### DISCUSSION THREAD

**Discussion by sfayer** [DIRACGrid/DIRAC#6374] [General]
Hi,

Now that we have the CloudCE code, it's probably time to think about removing the VMDIRAC stuff (e.g. CloudDirector and Resources) to finally close #5282.

@fstagni (and anyone else who has an opinion), do you have preference on which version we remove it in? I'm thinking of just doing it in the current integration (e.g. for 8.1) unless someone has a strong opinion otherwise.

I considered suggesting adding a deprecated tag to the execute function of the CloudDirector but I doubt anyone will really see it there, I think just announcing it in the next BiLD meeting should be sufficient.

@IgorPelevanyuk Are you still using VMDIRAC? I think you mentioned moving away from clouds but it wasn't clear if you meant in general or for a specific resource. If we need to add OpenNebula support to CloudCE that's something we can do, but it'll take a little while as it's a moderately big project.

Regards,
Simon

**Comment by fstagni** [comment]
Removing it from 8.1 is perfectly fine. An announcement to the BiLD and a not in the wiki will be enough.
